themes/sim-web.omp.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "right",
+      "filler": " ",
```

Comment:
why is there an empty filler?

Reply:
Sorry, did not run the correct command to generate the image, resulting in a style exception